static/app/components/objectInspector.tsx

```diff
@@ -37,17 +44,69 @@ function ObjectInspector({data, theme, ...props}: Props) {
     [isDark, theme, emotionTheme.red400, emotionTheme.text]
   );
 
-  return (
+  const [tooltipState, setTooltipState] = useState<'copy' | 'copied' | 'error'>('copy');
+
+  const tooltipTitle =
+    tooltipState === 'copy'
+      ? t('Copy')
+      : tooltipState === 'copied'
+      ? t('Copied')
+      : t('Unable to copy');
```

Comment:
shouldn't this return back to 'copy' after a timeout?

Reply:
it swaps back in the `onMouseLeave` callback

i copied it all from <CodeSnippet> which i noticed should probably use <Clipboard> because Codesnippet isn't checking for browser support.

https://github.com/getsentry/sentry/blob/feat-replay-add-method-column-network/static/app/components/codeSnippet.tsx#L72-L81

Reply:
Does it even need different tooltip states? The toast indicators pop up w/ different states

Reply:
I think it benefits from the basic 'copy' state. Otherwise it's hard to spot overall, when the tooltip pops up right away it stands out more.

But i just threw it in because i had the example open. Either way works for me

Reply:
Lets leave the whole pile in, and design can help on the next rev. I'm talking about making it visible/discoverable maybe there are other ways to solve that.